src/common/adapters/jira.test.js

```diff
@@ -76,6 +76,15 @@ describe('jira adapter', () => {
     expect(result).toEqual([ticket]);
   });
 
+
+  it('extracts tickets from new generation software projects', async () => {
+    const result = await scan(loc('my-subdomain.atlassian.net', `/jira/software/projects/TT/boards/8/backlog?selectedIssue=${key}`), doc);
```

Comment:
The `loc` test helper isn't really written in the best way (sorry 🙄), but for consistency, we should do as on line 62:

```suggestion
    const result = await scan(loc('my-subdomain.atlassian.net', '/jira/software/projects/TT/boards/8/backlog', `?selectedIssue=${key}`), doc);
```